pkg/protocol/uri_windows.go

```diff
@@ -53,3 +55,21 @@ func addLeadingSlash(dst, src []byte) []byte {
 
 	return dst
 }
+
+// checkSchemeWhenCharIsColon check url begin with :
+// Scenarios that handle protocols like "http:"
+// Add the path to the win file, e.g. "E:/gopath".
+func checkSchemeWhenCharIsColon(i int, rawURL []byte) (scheme, path []byte) {
+	if i == 0 {
+		hlog.Errorf("error happened when trying to parse the rawURL(%s): missing protocol scheme", rawURL)
+		return
+	}
+
+	if i+2 < len(rawURL) && rawURL[i+1] == '/' {
```

Comment:
What about rawURL is `E:/`? Seems not a good idea that return `nil,nil` in this scenario?